skimage/measure/tests/test_find_contours.py

```diff
@@ -45,6 +48,36 @@ def test_binary():
     assert_array_equal(contours[0][::-1], ref)
 
 
+def test_nodata():
+    ref = [[6. ,  1.5],
+           [5. ,  1.5],
+           [4. ,  1.5],
+           [3. ,  1.5],
+           [2. ,  1.5],
+           [1.5,  2. ],
+           [1.5,  3. ],
+           [1.5,  4. ],
+           [1.5,  5. ],
+           [1.5,  6. ],
+           [1. ,  6.5],
+           [0.5,  6. ],
+           [0.5,  5. ],
+           [0.5,  4. ],
+           [0.5,  3. ],
+           [0.5,  2. ],
+           [0.5,  1. ],
+           [1. ,  0.5],
+           [2. ,  0.5],
+           [3. ,  0.5],
+           [4. ,  0.5],
+           [5. ,  0.5],
+           [6. ,  0.5]]
+    contours = find_contours(b, 0.5, positive_orientation='high',
+                             nodata=np.nan)
+    assert len(contours) == 1
+    assert_array_equal(contours[0][::-1], ref)
```

Comment:
Could you perhaps change the order of `ref` above? Than you can drop the inversion here.

Reply:
As in the previous comment, this is done in the same way as the existing tests; but I agree with you that this one in particular is unnecessary.

I'd be happy to make your suggested tweak (for both this and the previous item), but I think that if I do, I should also make the same changes to the existing test `test_binary` - what do you think?